app/controllers/with_user_controller.rb

```diff
@@ -4,9 +4,16 @@ class WithUserController < ApplicationController
   before_action :authorize!
 
   def authorize!
-    # requested is authorized if using okta to provide a valid user id
-    return if authorization && current_user
+    # requested is authorized if using okta to provide a valid user id, and no specific user is set
+    return if authorization && current_user && !@user
+
+    # currently, if trying to operate on a specific @user, you can only operate on your own user data, but this may change later
+    return if @user && @user == current_user
 
     render_unauthorized
   end
+
+  def set_user(user_id_attribute = :user_id)
+    @user = params[user_id_attribute].blank? || params[user_id_attribute] == "me" ? current_user : User.find(params[user_id_attribute])
```

Comment:
So long as rubocop doesn't complain, I would appreciate parens here:
```suggestion
    @user = (params[user_id_attribute].blank? || params[user_id_attribute] == "me") ? current_user : User.find(params[user_id_attribute])
```

Reply:
Sorry, no such luck.

standard: Use Ruby Standard Style (https://github.com/testdouble/standard)
standard: Run `standardrb --fix` to automatically fix some problems.
  app/controllers/with_user_controller.rb:17:13: Style/TernaryParentheses: Omit parentheses for ternary conditions.

I like the look of parens here also.. :shrug: